### DISCUSSION TITLE
Does Swell Frontend Api support batch fetching.

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#447] [Help] [answered]
I was wondering swell supports batch fetching where specific product can be fetched using their Id's.
specifically for this api
await swell.products.get(productId);
if so the how would one implement a batch fetch for a list of Id's on this api.

Desperate for help and ideas

**Comment by ericingram** [comment, marked as answer]
You can fetch multiple products by ID at once, like this:
```
await swell.products.list({
  id: { $in: [productId1, productId2, productId3, ...]},
  limit: 10 // should be at least the number of IDs above or it will paginate
})
```

**Reply by Daniel-Alamezie** [reply]
does this work with the just the await swell.products.list api, or does that apply to the 
await swell.products.get api too?

**Reply by ericingram** [reply]
It's just for list, which returns multiple records, while get returns just 1.

**Comment by Daniel-Alamezie** [comment]
I ask because the specific information i need exists in this api await swell.products.get()
